open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/repositoryconnector/OMRSRepositoryHelper.java

```diff
@@ -845,6 +918,17 @@ EntityDetail incrementVersion(String              userId,
                                   InstanceAuditHeader originalInstance,
                                   EntityDetail        updatedInstance);
 
+    /**
+     * Changes the control information to reflect an update in an instance proxy
+     *
+     * @param userId            user making the change.
+     * @param originalInstance  original instance before the change
+     * @param updatedInstance   new version of the instance that needs updating
+     * @return updated instance
+     */
+    EntityProxy incrementVersion(String              userId,
```

Comment:
The version should never be incremented in an entity proxy because it is never editted - entity proxies are generated from the EntityDetail